### DISCUSSION TITLE
Add Steam's new runtime as a runner for Linux native games in containers

### DISCUSSION THREAD

**Discussion by VictorVow** [lutris/lutris#3374] [Ideas]
Steam [just released](https://steamcommunity.com/app/221410/discussions/0/1638675549018366706/) a new version of the Steam Runtime which supports containerisation for games and it's based off Flatpak technology as I understand. This should be available as a new runner option in Lutris. 

To briefly list some benefits of using containers for native games:
- It ensures games will work across all Linux distributions (including older ones) without there having to be separate installers, instructions or troubleshooting
- Games are more likely to continue working on your computer no matter what updates you do on your OS, since the packages are separate in the container 
- Better security model which isolates the application from the rest of your system 
- Very low overhead compared to a virtual machine but you still get the similar benefits of increased security and ability to run on any distribution

**Comment by tannisroot** [comment]
Not sure if we can actually use this but it's worth looking at for the next release, as we have some serious runtime problems.

**Comment by strycore** [comment]
Moving this to a Discussion until a technical solution is found.
